tron/commands/client.py

```diff
@@ -18,7 +18,7 @@
 
     assert simplejson  # Pyflakes
 except ImportError:
-    import json as simplejson
+    import json as simplejson  # type: ignore
```

Comment:
at some point we should replace all these simplejson usages with using/importing the stdlib version (since they should be the same thing)

but that's for another day!